cloudability/metrics-agent
# azure blob storage upload support

#### What does this PR do?
Adds support for uploading to a custom Azure blob storage

#### Where should the reviewer start?

#### How should this be manually tested?
Created an image based on this build and applied the image to a metrics-agent yaml. Deployed that metrics-agent to a test cluster.
Set all custom azure env vars with the correct values.
Confirm uploads to the Azure blob container name provided.

#### Any background context you want to provide?

#### What picture best describes this PR (optional but encouraged)?

#### What are the relevant Github Issues?

#### Developer Done List


#### For the Reviewer:

#### By approving this PR, the reviewer acknowledges that they have checked all items in this done list.

#### Reviewer/Approval Done List
